ssl-certificate-update.md

```diff
@@ -8,7 +8,7 @@ next: /patterns
 
 # SSL Certificate Updates
 
-**UPDATE 2014-12-21**: RubyGems 1.8.30, 2.0.15 and 2.2.3 have been released.
+**UPDATE 2016-10-06**: RubyGems 1.8.30, 2.0.17, 2.2.5, and 2.6.7 have been released.
```

Comment:
Can you add this update above the existing update? And add a line with something like "Please make sure you always download the latest released version"